exercises/list-ops/src/test/java/ListOpsTest.java

```diff
@@ -58,17 +58,29 @@ public void testConcatOnNonEmptyListOfLists() {
     @Test
     public void testConcatOnNonEmptyListOfNestedLists() {
         List<List<List<Character>>> listOfNestedLists = Arrays.asList(
-                Arrays.asList(Collections.singletonList('1'),
-                Collections.singletonList('2')),
-                Arrays.asList(Collections.singletonList('3'),
-                Collections.emptyList()),
-                Arrays.asList(Arrays.asList('4', '5', '6'));
-
-        assertEquals(
-                Arrays.asList(Collections.singletonList('1'),
-                Collections.singletonList('2'), Collections.singletonList('3'),
-                Collections.emptyList(), Arrays.asList(4, 5, 6)),
-                ListOps.concat(listOfNestedLists));
+                Arrays.asList(
+                        Collections.singletonList('1'),
+                        Collections.singletonList('2')
+                ),
+                Arrays.asList(
+                        Collections.singletonList('3'),
+                        Collections.emptyList()
+                ),
+                Arrays.asList(
+                        Arrays.asList('4', '5', '6')
+                )
```

Comment:
Unless I'm reading the [canonical data](https://github.com/exercism/problem-specifications/commit/ccdfa4af6b1d2187194d7b903188b8f706e596a4#diff-637e4f7a1434d17126301fcffc3bbcbe) wrong, I think this should actually be:
```
Arrays.asList(
    Collections.singletonList('1'),
    Collections.singletonList('2')
),
Collections.singletonList(
    Collections.singletonList('3'),
),
Collections.singletonList(
    Collections.emptyList()
),
Collections.singletonList(
    Arrays.asList('4', '5', '6')
)
```
What do you think? 🙂 

Also, I think the indentation is a little bit off, eveything should be indented with four spaces, as with the rest of the file 🙂